src/lib/caching/structures/guilds/GuildMember.ts

```diff
@@ -94,6 +94,19 @@ export class GuildMember extends Structure {
 		return this.nick ?? this.user?.username ?? null;
 	}
 
+	/**
+	 * The displayed color of the member
+	 * @since 0.0.4
+	 */
+	public get displayColor(): number {
+		let highestRole = null;
+		for (const role of this.roles.values()) {
+			if (!highestRole || role.position < highestRole.position) continue;
+			if (role.color) highestRole = role;
+		}
+		return highestRole ? highestRole.color : 0;
```

Comment:
this should return the hexadecimal
```suggestion
		return highestRole ? highestRole.color : 0x000000;
```